docs/en/operations/table_engines/join.md

```diff
@@ -1,36 +1,52 @@
-
 # Join
 
-A prepared data structure for JOIN that is always located in RAM.
+Prepared data structure for using in [JOIN](../../query_language/select.md#select-join) operations.
+
+## Creating a table
 
 ```
-Join(ANY|ALL, LEFT|INNER, k1[, k2, ...])
+CREATE TABLE [IF NOT EXISTS] [db.]table_name [ON CLUSTER cluster]
```

Comment:
are we always duplicating this for other table engines?

Reply:
I made it for the MergeTree family firstly, then we decided (if I'm not mistaken) to spread it to other engines. All the engines docs that I have changed up to this time correspond to this template.